.infra/application.properties

```diff
@@ -27,4 +27,7 @@ debezium.transforms.ReadOperationFilter.condition=!(valueSchema.field('op') && v
 debezium.transforms.PostsFilter.type=io.debezium.transforms.Filter
 debezium.transforms.PostsFilter.language=jsr223.groovy
 debezium.transforms.PostsFilter.condition=!(valueSchema.field('op') && value.op == 'u' && value.source.table == 'post' && value.before.views != value.after.views)
+debezium.transforms.SourceFilter.type=io.debezium.transforms.Filter
+debezium.transforms.SourceFilter.language=jsr223.groovy
+debezium.transforms.SourceFilter.condition=!(valueSchema.field('op') && value.op == 'u' && value.source.table == 'source' && value.before.flags != value.after.flags)
```

Comment:
Wondering if we could replace `value.before.flags != value.after.flags` with `valueSchema.field('after.flags')`? Since we want to exclude all flags?